docs/companion_guide.md

```diff
@@ -411,6 +411,31 @@ billion transactions per second (tps).
 | Schema / Data Representation | JSON Formatted |
 | Query Capabilities | [Not found in documentation] |
 
+| Peergos	   |  https://https://peergos.org/ |
+| -------------- | ---------------------------------------------------------- |
+| Description	   | Peergos is a decentralised protocol and open-source platform for storage, social media and applications |
+| Specification  | Proprietary specification |
```

Comment:
What is meant by proprietary here? Peergos protocol is 100% open.

Reply:
@ianopolous happy to change it ( or would welcome a PR to update it if you'd be willing ). Could you please point to the specification so we can make the update?